src/transaction/log_tran_table.c

```diff
@@ -1588,6 +1588,14 @@ logtb_clear_tdes (THREAD_ENTRY * thread_p, LOG_TDES * tdes)
   LSA_SET_NULL (&tdes->rcv.atomic_sysop_start_lsa);
   LSA_SET_NULL (&tdes->rcv.analysis_last_aborted_sysop_lsa);
   LSA_SET_NULL (&tdes->rcv.analysis_last_aborted_sysop_start_lsa);
+
+  /* reflect the session (or prm_Def) */
+  tdes->isolation = (TRAN_ISOLATION) prm_get_integer_value (PRM_ID_LOG_ISOLATION_LEVEL);
```

Comment:
Does the following case work well?
```
# in the session-1: csql
change isolation_level = 6 # serializable
execute transaction and commit

# in the session-2: csql
change isolation_level = 4 # read committed
execute transaction and commit

# in the session-1: csql
execute transaction # this transaction work serializable but I worry about it work read committed
```

Reply:
@hornetmj 

It seems to be working fine.

session-1 (csql)
```
csql> set system parameters 'isolation_level=6';
csql> ;autocommit off
csql> select * from tbl; ------- (1)

csql> select * from tbl; ------- (3)

csql> commit;
csql> select * from tbl;
```
session-2 (csql)
```
csql> insert into tbl values (20); ------- (2)
```

I used the above test.

(1) and (3) show the same result without `20`. And i can see a value `20` inserted by session 2 after a commit.